library/src/main/java/com/xwray/groupie/GroupAdapter.java

```diff
@@ -107,30 +98,44 @@ public void updateAsync(@NonNull final List<? extends Group> newGroups) {
      *
      * @param newGroups List of {@link Group}
      * @param onAsyncUpdateListener Optional callback for when the async update is complete
+     * @param detectMoves Boolean is passed to {@link DiffUtil#calculateDiff(DiffUtil.Callback, boolean)}
      */
     @SuppressWarnings("unused")
-    public void updateAsync(@NonNull final List<? extends Group> newGroups, @Nullable final OnAsyncUpdateListener onAsyncUpdateListener) {
+    public void updateAsync(@NonNull final List<? extends Group> newGroups,
+                            @Nullable final OnAsyncUpdateListener onAsyncUpdateListener,
+                            boolean detectMoves) {
```

Comment:
can you change the signature so the `OnAsyncUpdateListener` is the last function parameter - it's better for trailing lambda usages in Kotlin.

Reply:
Changed it 👍